tap_github/repository_streams.py

```diff
@@ -1953,3 +1954,54 @@ def _write_schema_message(self):
         if not self._schema_emitted:
             super()._write_schema_message()
             self._schema_emitted = True
+
+
+class DependentsStream(GitHubRestStream):
+    """Defines 'dependents' stream."""
+
+    name = "dependents"
+    path = "/{org}/{repo}/network/dependents"
+    primary_keys = ["repo_id", "dependent_name_with_owner"]
```

Comment:
Do I need to also create `dependent_name_with_owner` at the top level or is the current implementation ok @edgarrmondragon ? (it's passed as `{ dependent: { name_with_owner } }`

Reply:
@laurentS do you know?

Reply:
I don't know if this applies to all targets, but `target-postgres` translates nested fields as `dependent__name_with_owner` (note the double underscore), but I'm not sure if this is part of the signer spec, or just valid for this particular target.

Reply:
Ok, to be safe I'll extract it as an independent column then